Dockerfile

```
@@ -0,0 +1,10 @@
+FROM node:latest
+WORKDIR /app
+COPY package*.json ./
+RUN npm install
+RUN npm install -g nodemon
```

Hadolint

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)